rolling_update.yml

```diff
@@ -59,7 +63,7 @@
       until: result.rc == 0
       retries: 5
       delay: 10
-      delegate_to: 127.0.0.1
+      delegate_to: "{{ groups.mons[0] }}"
```

Comment:
Not sure about this one, using the first monitor is not that good if it's the one that has been or is been updated